third_party/move/move-model/src/exp_rewriter.rs

```diff
@@ -457,24 +456,39 @@ pub trait ExpRewriterFunctions {
             Match(id, disc, arms) => {
                 let (id_changed, new_id) = self.internal_rewrite_id(*id);
                 let (disc_changed, new_disc) = self.internal_rewrite_exp(disc);
+
```

Comment:
nit: unintended newline?

Reply:
I think having an empty line before the section dealing with arms made the code clearer, so I'm leaving it.